Lib/_pyrepl/console.py

```diff
@@ -16,6 +16,7 @@
 # RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF
 # CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
 # CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+import __future__
 
 from __future__ import annotations
```

Comment:
```suggestion

from __future__ import annotations
import __future__
```

Can we switch it like this?

Reply:
After my testing, no. If switches it, the code will be invalid. I don't know why
![2024-10-05 190103](https://github.com/user-attachments/assets/0f7563a8-07ec-4438-99ba-7f3e42592de5)

Reply:
Maybe a compiler/parser issue. Anyway, instead of import `__future__` like this, you can import it locally using `__import__("__future__", fromlist=["barry_as_FLUFL"]).barry_as_FLUFL`. You only want the flag's bit.